### DISCUSSION TITLE
Wake Induced Mixing Gain and turbulence intensity for floating farms

### DISCUSSION THREAD

**Discussion by fbelli997** [NatLabRockies/floris#1050] [Q&A]
Hello everyone,
I was approaching the calculation of the AEP with the wake-induced mixing gain to set the increase in wake amplitude, and I noticed this warning message:
"floris.logging_manager.LoggingManager WARNING Running wake_induced_mixing model with mixing contributions from the atmospheric turbulence intensity has not been vetted. To avoid this warning, set atmospheric_ti_gain=0. in the FLORIS input yaml."

So, is it currently not possible to perform tuning on the wake-induced mixing in the case of floating farms?
Additionally, is there some kind of relationship between the wake-induced mixing gain parameters and the classical turbulence intensity input parameter, or do I need to calibrate the parameter to match the wake deficit as a function of the x-coordinate?

Thanks in advance, everyone!

**Comment by misi9170** [comment]
Hi @fbelli997 ,

Thanks for you question! If I'm understanding correctly, you're running into this warning because you've set the `atmospheric_ti_gain` term in the `wake_induced_mixing` model parameters to something other than zero. 

The form of the wake induced mixing (WIM) model is given [here in the documentation](https://nrel.github.io/floris/empirical_gauss_model.html#wake-induced-mixing) (you've probably already seen this, but I just wanted to be sure). As you've probably seen, we don't have current documentation for how the `atmospheric_ti_gain` affects this (and the documented model assumes that `atmospheric_ti_gain = 0.0`). Setting a nonzero value for `atmopsheric_ti_gain` _does_ however affect the WIM---see [these lines](https://github.com/NREL/floris/blob/fd37e9b68fefe766cc5b989f8a8f40602913170e/floris/core/solver.py#L1180-L1188) of the Empirical Gaussian (EmG) model solver procedure. However, we have in no way tested that dependency and moreover, we may change the functional form of the dependency in future; finally, the other parameters of the EmG model were tuned with `atmospheric_ti_gain` set to zero. This is why we raise a warning if users change the value away from zero.

We do have some long-term plans to develop the dependency of the WIM model to TI (see #841). However, there isn't a current timeline for that work, and it is unlikely to happen in the next 6 months or so (and quite possibly not for longer than that). 

That being said, you are welcome to try tuning the `atmospheric_ti_gain`---we'd certainly be interested in what you find! Just be aware that you may need to retune other parameters of the EmG model (likely the `wake_expansion_rates` and `mixing_gain_velocity` when you do this). If you are doing this and would like to silence the warning being raised, you can set `logging > console > enable` to `False` in the main input yaml file.

Best,
Misha
